### PR TITLE
 adding Ebuka-Chibueze to team.md

### PR DESCRIPTION
**Pull request by Ebuka-Chibueze**

### CODE REVIEW

**Comment by hikaruit15** [comment]
Hi @Ebuka-Chibueze,
When you become an intern, you will be added to team.md.
It means you have to finish the first steps.

**Comment by Ebuka-Chibueze** [comment]
Hmm, I'm a bit confused. I completed each step including the survey. Step 9 says to add yourself to the virtual intern list after completing the survey.

**Comment by dogi** [comment]
@Ebuka-Chibueze @hikaruit15 sorry for the confusion - I am hoping that soon you will 5 accepted PRs so this PR here will also be a go

@dogi that did already missing things on step7
`ebuka-chibueze	02/19/2017	0.12.81	02/19/2017	02/19/2017	2	0`

I also have already your survey ... but there are 2 of the 5 PRs still in the reviewing process

@Ebuka-Chibueze did I sum up all nicely? or did I forget something?

**Comment by Ebuka-Chibueze** [comment]
Oh I see what you mean now.

File: pages/vagrant.md.orig [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Vagrant
```
Comment (dogi) [inline pages/vagrant.md.orig]: please remove this page

**Pull request merged**